api/cpp/include/slint_platform.h

```diff
@@ -227,6 +227,53 @@ class SoftwareRenderer
     }
 };
 
+/// An opaque, low-level window handle that internalizes everything necessary to exchange messages
+/// with the windowing system. This includes the connection to the display server, if necessary.
+///
+/// Note that this class does not provide any kind of ownership. The caller is responsible for
+/// ensuring that the pointers supplied to the constructor are valid throughout the lifetime of the
+/// WindowHandle.
+class WindowHandle
```

Comment:
i'd maybe call that `NativeWindowHandle`

Reply:
Good idea, thanks. Will rename.